Example/HorizonCalendarExample/HorizonCalendarExample/Demo View Controllers/DemoViewController.swift

```diff
@@ -50,8 +50,8 @@ class DemoViewController: UIViewController {
     switch monthsLayout {
     case .vertical:
       NSLayoutConstraint.activate([
-        calendarView.topAnchor.constraint(equalTo: view.layoutMarginsGuide.topAnchor),
-        calendarView.bottomAnchor.constraint(equalTo: view.layoutMarginsGuide.bottomAnchor),
+        calendarView.topAnchor.constraint(equalTo: view.topAnchor),
+        calendarView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
```

Comment:
now that this bug is fixed, we can constrain to the top / bottom anchor and let the system set our `layoutMargins`. Scroll-to-item will work with no issues.